### ISSUE TITLE
Arrest Reports not staying in system?

### ISSUE THREAD

**Issue by newm-newm** [PieRGud/ComputerPlus#31]
Any arrest report or traffic report that is filled is immediately gone when running license/vehicle again. Only shows up while the vehicle/license window is up.

**Comment by ainesophaur** [comment]
The reports and citations are actually being saved, but the lookup query
was causing too many crashes. I'll have to replace the query building
library with something custom. Once that's fixed the previous reports will
appear.

Just need the time to do so.

**Comment by ainesophaur** [comment]
Also if you're a developer I could use some extra hands. Work has had me
pegged to a wall so if someone wants to take a stab at it I can point them
in the right direction

**Comment by newm-newm** [comment]
I can attempt it, never coded in C# though.

**Comment by ainesophaur** [comment]
It's all good. Let's hope for a week of no 0 day exploits at work so I can
reclaim my weekend

**Comment by ItaLiaNKiinG** [comment]
I'm willing to pay top dollar for the time and effort to get the computer to save arrest/citation reports..

**Comment by ainesophaur** [comment]
That isn't necessary. I need time, not money. Trust me its something I'm dying to finish, but other things take priority

**Comment by ItaLiaNKiinG** [comment]
If I hire a professional can you tell me which files he needs to work on and I'll have someone do it for you?

**Comment by ainesophaur** [comment]
It's not a simple file to change.. Its replacing an entire module that is currently provided by a third party library. I'd have to sit down and brain dump where everything is at in order to outline what's needed, and by that time I could just make the modifications myself.

**Comment by ItaLiaNKiinG** [comment]
Wouldn't a professional be able to figure it out? I'm sure if I tell him what needs to be done he could be able to get it done, right?

**Comment by ItaLiaNKiinG** [comment]
I'm willing to spend 200+ dollars to get it done.

**Comment by Stealth22PD** [comment]
You do know that he _is_ a developer for a living, right?

**Comment by ItaLiaNKiinG** [comment]
No, I didn't know he does it for a living but since he's finding it hard to make time for this fix I'm willing to hire a coder myself to get this done a little faster.. like I said I'm willing to spend $200+ for the time and effort to expedite this fix

**Comment by ItaLiaNKiinG** [comment]
It will really complete the LSPDFR experience for me which is why I'm willing to spend actually money to get this fixed

**Comment by ainesophaur** [comment]
It would complete the experience for everyone.

This isn't something that is solved by throwing money at it. Ultimately you could hire someone to complete it.. But let's say they decide to add four new dependencies on other libraries.. I can tell you now I wouldn't accept their code and merge it into the base. This is a project that was developed for the improved experience for all players and I know this is one of the most important things to complete.. However, it was called a beta release for a reason.. It requires time and a thought out solution that doesn't add additional overhead

**Comment by ItaLiaNKiinG** [comment]
Well if you do allow me to hire someone to complete it you make the rules, if they don't do it the exact way you want it done they don't get paid, simple as that. I would be paying them $25 - $50 as insurance before they start on it so they can rest assured that they will get their money, but they have to accept the parameters first obviously.

**Comment by PieRGud** [comment]
You can fork the repo and pay whoever you want. We don't have the time to make rules. As ainesophaur already said, development is done in his free time. If you want an immediate solution, then we can't help you. That's not how this works.

**Comment by ItaLiaNKiinG** [comment]
Okay, I understand.. hey quick favor, can anyone here check my crash report and figure out why my LSPDFR keeps crashing?

**Comment by ItaLiaNKiinG** [comment]
Nvm.. too long.

**Comment by ItaLiaNKiinG** [comment]
Do you guys have a rough estimation on when you'll be fixing the problem?

**Comment by ItaLiaNKiinG** [comment]
Sorry for bugging you but do you have a rough estimate of when you'll be fixing the problem?

**Comment by newm-newm** [comment]
https://www.lifehacker.com.au/2016/04/github-now-lets-you-block-troublesome-users-from-public-repositories/

**Comment by ainesophaur** [comment]
@ItaLiaNKiinG it'll be done when its done.. there is no estimate of when I will start nor how long it will take. My life responsibilities take precedence and I do not have the capacity to work on it until I have a break of freedom. If you want to whisper sweet nothings into my servers at work, it'd probably help.. I can't forecast when shit breaks or when I'll be working 125+ hours in one week. By constantly asking me to provide you with an estimate, I feel less and less inclined to work on it. Please be patient.. everyone wants the feature, not just you. You're tarnishing the community feeling of this plugin by being up my ass every time you think about it.

**Comment by ItaLiaNKiinG** [comment]
It was just a question chill the fuck out

**Comment by PieRGud** [comment]
Blocked @ItaLiaNKiinG.

**Comment by BejoIjo** [comment]
Hi,

I made some crash-bug fixings and enable the arrest report / citation history. So far I cannot play with it for hours without crashing.

Let me know if I can contribute.

BR.

**Comment by newm-newm** [comment]
@BejoIjo submit your changes in a pull request.

**Comment by ainesophaur** [comment]
He already has an open pr waiting approval and merge.
